hack/verify_pipeline.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -ue
+
+# Use the policies defined in the Enterprise Contract to validate a 
+# pipline definition
+#
+# Tyipcal usage:
+#  oc get pipeline somePipeline -n someNamespace -o jsonpath='{.items[0]}' > pipeline.json
```

Comment:
This doesn't work for me. Am I missing something?:

```
$  oc get pipeline buildah -n openshift -o jsonpath='{.items[0]}'
# nothing...
```
This is what I was expecting we'd use, e.g.:
```
$  oc get pipeline buildah -n openshift -o json | head -3
{
    "apiVersion": "tekton.dev/v1beta1",
    "kind": "Pipeline",
```

Reply:
If I use:
` oc get pipeline -n jvm-build-service -o jsonpath='{.items[0]}' > /tmp/foo.json`
I get:
```
{
  "apiVersion": "tekton.dev/v1beta1",
  "kind": "Pipeline",
  "metadata": {
```

Perhaps the `oc` command specifics will depend whether the returned json has a top level array.